sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -161,7 +161,7 @@ static pi_result USMSetIndirectAccess(pi_kernel kernel) {
 extern "C" {
 
 // Convenience macro makes source code search easier
-#define OCL(pi_api) Ocl##pi_api
+#define OCL(pi_api) pi_api
```

Comment:
I don't understand the purpose of this change, but I think this macro should be just deleted.

Reply:
@bader, the purpose is to expose pi* symbols from the plugin and not Oclpi* ones. Of course the OCL macro is now redundant.

Reply:
What was the motivation to have `Ocl` prefix?

Reply:
It was needed at the times when we linked with plugins code, not dlopen()-ed them.